py/torch_tensorrt/dynamo/_defaults.py

```diff
@@ -28,6 +28,7 @@
 DRYRUN = False
 HARDWARE_COMPATIBLE = False
 SUPPORTED_KERNEL_PRECISIONS = {dtype.f32, dtype.f16, dtype.bf16, dtype.i8, dtype.f8}
+TIMING_CACHE_PATH = "/tmp/timing_cache.bin"
```

Comment:
Could you use a more cross-platform default path because `/tmp` is not a valid path on Windows

Reply:
I modified it to use `tempfile.gettempdir()` now which gives `C:\\Users\\GSTEFA~1\\AppData\\Local\\Temp` on windows and `/tmp` on linux